fix: `base_url` becomes empty after saving configuration

yzqzss opened the pull request:
Changes proposed in this pull request:

Since #5657 sets the base URL input box to `disabled="disabled"`, the browser will not send the `base-url` parameter. As a result, when the submit button is clicked, the base URL will be cleared.

How to test the feature manually:

before:

https://github.com/FreshRSS/FreshRSS/assets/30341059/a9760aca-8742-4668-9f43-9ac0fc72dbe8

fixed:

https://github.com/FreshRSS/FreshRSS/assets/30341059/70e1f3f2-d1de-402f-b40f-f40634311f62


Pull request checklist:

- [x] clear commit messages
- [x] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

yzqzss committed:
fix: `base_url` becomes empty after saving configuration

Frenzie commented:
Fwiw there's also the `readonly` attribute for `<input>` elements.

Frenzie reviewed (approved):
Seems sensible enough.

Alkarex committed:
Just remove the line

Alkarex commented:
Regression from https://github.com/FreshRSS/FreshRSS/pull/5657

> Fwiw there's also the `readonly` attribute for `<input>` elements.

Indeed @Frenzie , I was not aware of the difference in POST behaviour

https://developer.mozilla.org/docs/Web/HTML/Attributes/readonly#attribute_interactions

Alkarex committed:
Use read-ony instead
https://developer.mozilla.org/docs/Web/HTML/Attributes/readonly#attribute_interactions

Alkarex committed:
Revert for websub

Alkarex committed:
Changelog

Alkarex commented:
Well spotted @yzqzss , thanks!
